help/en/html/tools/StringIO.shtml

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta name="generator" content="HTML Tidy for HTML5 for Apple macOS version 5.8.0">
+  <title>JMRI: StringIOs</title>
+  <!--#include virtual="/help/en/parts/Style.shtml" -->
+</head>
+<body>
+  <!--#include virtual="/help/en/parts/Header.shtml" -->
+
+  <div id="mBody">
+    <!--#include virtual="Sidebar.shtml" -->
+
+    <div id="mainContent">
+      <h1>JMRI: StringIO Documentation</h1>
+      A JMRI StringIO object usually connects to some ability of a hardware
+      system to get and put strings.
+      <h2>The StringIO Table</h2>
+      The <a href="../../package/jmri/jmrit/beantable/StringIOTable.shtml">StringIO Table</a> lets you
+      see what StringIOs have been defined,
+      what value they currently contain, and to send new values to the layout. 
+      You can type a new string value into a
+      StringIO.
+      <h4>StringIO Table controls</h4>
+
+      <ul>
+        <li>Below the StringIO Table you'll find the <a href=
+        "../../package/jmri/jmrit/beantable/StringIOAddEdit.shtml">Add...</a> button to add new
+        Memories.
+        </li>
+      </ul>
+
+      <h2>Using StringIOs with Logix</h2>
+      A Logix can test a StringIO for particular contents, and set the contents of a StringIO to
```

Comment:
This should be LogixNG, not Logix.

Reply:
Thanks!  Fixed by changing to "Logix or LogixNG" there and in the Memories page.